<?xml version="1.0"?>
<launch>
	<node pkg="actuator_controller" type="actuator_controller" name="actuator_controller" output="screen" > 
        	<rosparam command="load" file="$(find actuator_controller)/yaml/actuator_controller.yaml" />
		<param name="cmdTopicName" type="string" value="/cmd"/>
		<param name="stateTopicName" type="string" value="/state"/>
		<param name="serverIP" type="string" value="169.254.1.1" />
		<param name="pubPos" type="bool" value="true" />
		<param name="pubVel" type="bool" value="true" />
		<param name="MaintenanceMode" type="bool" value="false" />
	</node>	

	<!-- Start up the dynamic reconfigure GUI. -->
  	<!--<node
    		pkg="dynamic_reconfigure" type="reconfigure_gui" name="reconfigure_gui">
  	</node> -->
</launch>



